### PR TITLE
Make examples section nicer.

### PR DESCRIPTION
**Pull request by Rowlando13**
Makes the example section nicer by including a bring explanation. Still include the screen cast, with comment that it is older.

### CODE REVIEW

**Comment by Rowlando13** [comment]
@saroad2  These are just doc changes and are good to go. They fail CI due to existing flake8 issues in the code base.

File: docs/quickstart.rst [hunk 1]
```diff
@@ -64,38 +64,26 @@ virtualenv::
...
 A few seconds later and you are good to go.
 
-Screencast and Examples
+Examples
 -----------------------
 
-There is a screencast available which shows the basic API of Click and
-how to build simple applications with it.  It also explores how to build
-commands with subcommands.
+Some stand alone examples of Click applications are packaged with Click. They are available in the `examples folder <https://github.com/pallets/click/tree/main/examples>`_ of the repo.
```
Comment (tachyondecay) [inline docs/quickstart.rst:70]: "standalone" is one word.

File: docs/quickstart.rst [hunk 2]
```diff
@@ -64,38 +64,26 @@ virtualenv::
...
+*   `colors <https://github.com/pallets/click/tree/main/examples/colors>`_ : A simple example that colorizes text. Uses colorama on Windows.
+*   `aliases <https://github.com/pallets/click/tree/main/examples/aliases>`_ : An advanced example that implements :ref:`aliases`.
+*   `imagepipe <https://github.com/pallets/click/tree/main/examples/imagepipe>`_ : A complex example that implements some :ref:`multi-command-chaining` . It chains together image processing instructions. Requires pillow.
+*   `repo <https://github.com/pallets/click/tree/main/examples/repo>`_ : An advanced example that implements a Git-/Mercurial-like command line interface.
+*   `complex <https://github.com/pallets/click/tree/main/examples/complex>`_ : A very advanced example that implements loading subcommands dynamically from a plugin folder.
+*   `termui <https://github.com/pallets/click/tree/main/examples/termui>`_ : A simple example that showcases terminal UI helpers provided by click.
+
+If you prefer video, an older screencast (2014) is available. It shows shows the basics of Click, how to build simple applications with it, and how to build commands with subcommands.
```
Comment (tachyondecay) [inline docs/quickstart.rst:82]: Should we even still be linking to a 10-year-old video, even if a lot of it is still up to date? There are plenty of third-party video tutorials out there that people can find by searching. I’d suggest removing this entirely.

**Comment by Rowlando13** [comment]
Make changes as @tachyondecay suggested.

**Pull request merged**